extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/JacksonProcessor.java

```diff
@@ -120,18 +120,18 @@ void register(
             BuildProducer<ReflectiveMethodBuildItem> reflectiveMethod,
             BuildProducer<AdditionalBeanBuildItem> additionalBeans) {
         reflectiveClass.produce(
-                new ReflectiveClassBuildItem(true, false,
-                        "com.fasterxml.jackson.databind.ser.std.SqlDateSerializer",
+                ReflectiveClassBuildItem.builder("com.fasterxml.jackson.databind.ser.std.SqlDateSerializer",
                         "com.fasterxml.jackson.databind.ser.std.SqlTimeSerializer",
                         "com.fasterxml.jackson.databind.deser.std.DateDeserializers$SqlDateDeserializer",
                         "com.fasterxml.jackson.databind.deser.std.DateDeserializers$TimestampDeserializer",
-                        "com.fasterxml.jackson.annotation.SimpleObjectIdResolver"));
+                        "com.fasterxml.jackson.annotation.SimpleObjectIdResolver").methods(true).build());
```

Comment:
For next time, you can even use the `methods()` shortcut.

Reply:
Yeah, I saw that, but honestly it feels weird to me :)